fs/rc/jobs/job_test.go

```diff
@@ -377,6 +377,8 @@ func TestRcJobList(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, out1)
 	assert.Equal(t, []int64{1}, out1["jobids"], "should have job listed")
+	assert.Equal(t, 1, out1["running"], "should have running job")
+	assert.Equal(t, 0, out1["finished"], "should not have finished job")
```

Comment:
This test seems to think `running` and `finished` are counts, but your `Stats` function returns lists of IDs.

Reply:
indeed
I fixed it